src/employeeProfiles.js

```diff
@@ -167,15 +169,37 @@ export const EmployeeProfile = () => {
             <TextField label="" source="personalEmail" />
           </SimpleShowLayout>
         </Grid>
-        {/*
-      <Grid item>
-        <SimpleShowLayout divider={<Divider flexItem />}>
-          <TextField label="Current assigment" source="" />
-          <DateField label="Hired Date" source="" />
-        </SimpleShowLayout>
-      </Grid>
-      Hidden empty fields until developed
-      */}
+        <Grid item>
+          <FunctionField
+            label=""
+            render={(record) => (
+              <SimpleShowLayout divider={<Divider flexItem />}>
+                {!record.startDate && (
+                  <span>This employee does not have any contracts</span>
```

Comment:
minor: i would change to:  <span> - </span>, what do you think guys?

Reply:
![image](https://github.com/entropy-code/entropay-admin-ui/assets/96883646/e06dfd2f-b0dd-4859-9144-01506ef95388)
Something like this?

Reply:
I would keep the "End Date" title and try to center the "-"